src/ResourceManager/Resources/Commands.Resources/Models.ResourceGroups/ResourceClient.Events.cs

```diff
@@ -1,104 +0,0 @@
-﻿// ----------------------------------------------------------------------------------
```

Comment:
so, this *should* be used in resource template deployment

Reply:
I didn't comment this out. I just deleted it since the entire file was commented out.